README.md

```diff
@@ -1,4 +1,52 @@
-## Setup the Database
+# Running the App
+The repository contains Docker configuration which can be used out of the box, however, the app can be run without using
+the docker container. Both way are discussed below:
+
+## 1. Run inside Docker Container
+It is the recommended way, the Docker Image of the app is pre-configured to run the `migrations` and `collectstatic`.
+However, the `load` command to populate the database and the crons need to be run manually.
+
+The image can be built locally or can be pulled from Docker Hub:
+
+```
+docker pull arjunsalyan/macports-webapp
+```
+or build the image
+```
+docker build -t macports-webapp .
+```
+
+After the image has been build or pulled, run it using an an env file that contains environment variables. Create a file
+`env` in the root of the project and supply the following information.
+
+**env**: *(Contains environment variables)*
+```
+DB_NAME=
+DB_USER=
+DB_PASSWORD=
+DB_HOST=
+SECRET_KEY=
```

Comment:
We could prepare `env.sample` and ask users to copy that file and fill in the blanks. It's ok to keep the variables documented here as they are now.

We need to document how to run with a local database running on the host machine.

Reply:
Thank you. I will add the file.

Once, I am finish collecting the bits about the entire load and update process, I will prepare a thorough documentation to run the docker image.

Reply:
I have done some changes to the README file. I think it would make sense to add more instructions in the other PR.

Reply:
Actually, I'd recommend adding `PORT` to be configured using environment variables as well, thereby completely removing the need to change anything in `settings`, am I right? 
Port is really a part of the host.

Reply:
Okay, thanks. Makes sense to put port in `env`.